# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatform.h
# Commit: 05896dbeb434
# Full Hash: 05896dbeb434d6f3ca9324f01eb67020d932f50f
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Determine webgl's layers::TextureType in Present. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79617
# ==============================================================================

diff -r 8439f89e170b -r 05896dbeb434 gfx/thebes/gfxPlatform.h
--- a/gfx/thebes/gfxPlatform.h	Mon Jun 15 18:26:12 2020 +0000
+++ b/gfx/thebes/gfxPlatform.h	Mon Jun 15 18:26:14 2020 +0000
@@ -771,6 +771,8 @@
   virtual void EnsureDevicesInitialized(){};
   virtual bool DevicesInitialized() { return true; };
 
+  virtual bool UseWaylandDMABufWebGL() { return false; }
+
   static uint32_t TargetFrameRate();
 
  protected: